src/appleseed/renderer/kernel/rendering/oiioerrorhandler.cpp

```diff
@@ -56,21 +56,18 @@ void OIIOErrorHandler::operator()(int errcode, const string& msg)
 
     switch (errcode & ErrorCodeMask)
     {
+      case EH_MESSAGE:
+      case EH_INFO:
+        RENDERER_LOG_DEBUG("%s", modified_msg.c_str());
+        break;
+
       case EH_WARNING:
         RENDERER_LOG_WARNING("%s", modified_msg.c_str());
         break;
 
       case EH_ERROR:
-        RENDERER_LOG_ERROR("%s", modified_msg.c_str());
-        break;
-
       case EH_SEVERE:
-        RENDERER_LOG_FATAL("%s", modified_msg.c_str());
```

Comment:
Why not using log_fatal ?

Reply:
`LOG_FATAL()` is pretty drastic. In appleseed.cli (the command line interface to appleseed) it will actually _terminate_ execution. Unless proven otherwise, I think it's safe to assume that no OSL message deserves such radical treatment.

Reply:
Ok sure :)